.yarn/plugins/plugin-git-hooks.js

```diff
@@ -0,0 +1,18 @@
+const { execSync } = require('child_process');
+
+const afterAllInstalled = (project) => {
+  execSync('yarn simple-git-hooks', { cwd: project.cwd, stdio: 'inherit' });
```

Comment:
Shouldn't we do this the same way we do it here? https://github.com/LavaMoat/LavaMoat/blob/main/packages/yarn-plugin-allow-scripts/sources/index.ts#L7

Reply:
This doesn't seem to work for local plugins:
```
Run yarn --immutable
Internal Error: Cannot find module '@yarnpkg/shell'
Require stack:
- /home/runner/work/key-tree/key-tree/.yarn/plugins/plugin-git-hooks.js
```